src/useBreakpoints.ts

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+
+// tailwind breakpoints
+const breakpoints = {
```

Comment:
As this is a constant, is probably better have it as UPPERCASE. But more a nit, honestly.